### ISSUE TITLE
reload quest_template_addon command

### ISSUE THREAD

**Issue by Aokromes** [TrinityCore/TrinityCore#29856]
### Description

we need to allow reload quest_template_addon 

### Expected behaviour

quest_template_addon is reloaded

### Steps to reproduce the problem

feature request

### Branch

3.3.5

### TC rev. hash/commit

ebdcb62 

### Operating system

n/r

### Custom changes

m/r

**Comment by Jonne733** [comment]
@Aokromes 
Sup dude, please correct me if I'm completely wrong in this assumptions or if I have missed something obvious, but isn't the command ".reload quest_template", when used in game,  something that reloads all of does tables that you mentioned.

**Comment by Aokromes** [comment]
hmmm not sure at least it doesn't shows reloading those tables.

**Comment by Jonne733** [comment]
Yes, that is true. A visual indication in the chat window that tells you that does tables also were reloaded would be handy.  😉

**Comment by Rushor** [comment]
sObjectMgr->LoadQuests();

handles the reload of all quest stuffs,

just change the text here:
https://github.com/TrinityCore/TrinityCore/blob/3.3.5/src/server/scripts/Commands/cs_reload.cpp#L548
